src/constants/java/net/fabricmc/yarn/constants/WorldEvents.java

```diff
@@ -0,0 +1,437 @@
+package net.fabricmc.yarn.constants;
+
+public final class WorldEvents {
```

Comment:
I hope you can add javadoc links to methods that actually process world events, like `net.minecraft.world.WorldAccess#syncWorldEvent(net.minecraft.entity.player.PlayerEntity, int, net.minecraft.util.math.BlockPos, int)`